input/fsh/SD_Genomics.fsh

```diff
@@ -235,20 +235,22 @@ Description:    "The area of the genome region referenced in testing for variant
 * component[genomicReferenceSequenceId] ^definition = "Range(s) of DNA sequence examined. The genomic reference sequence is a contiguous stretch of chromosome DNA that spans all of the exons of the gene and includes transcribed and non transcribed stretches. For this ID use either the NCBI genomic nucleotide RefSeq IDs with their version number (see: NCBI.NLM.NIH.Gov/RefSeq) or use the LRG identifiers, without transcript (t or p) extensions -- when they become available. (source: LOINC)"
 * component[genomicReferenceSequenceId].code = LNC#48013-7
 
-// * insert ReduceText
-// * insert ReduceText(referenceRange)
-// * insert ReduceText(component)
-// These are the additional MS on top of US Core Lab Observation
 * component and component.code and component.value[x] and component.dataAbsentReason MS
 
+Profile:    GenomicDiagnosticImplication
+Parent:     USCoreObservationLab
```

Comment:
GenomicDiagnosticImplication doesn't seem like an Observation. Would you say that knowledge about the impact and actionability of a variation exists outside the context of a patient? Did you consider any alternatives?

Reply:
this was added to align with the Clinical Genomics Reporting IG and was treated as a related observation on a variant: http://build.fhir.org/ig/HL7/genomics-reporting/StructureDefinition-diagnostic-implication.html. That profile 
contains more supporting components which I didn't include into mCODE but are relevant to further describing related phenotypes related to the variant.
CGWG applies the same approach to Pharmacogenomics and therapeutic implications.
I agree with their approach overall.
Did you have any thoughts on what that would be instead of an Observation or the CGWG approach overall?

Reply:
Maybe. agreed that it doesn't help in the short term but it's a valid perspective.
Perhaps we can align with the CG Reporting IG approach for now (especially since that has the approval of several organizations, including Epic _and_ Cerner representatives) and consider entering in a JIRA ticket. Reason is that what you're proposing is a major and significant change to the overall paradigm of handling groups of implications. The organization of a genomics report is pretty complex where there needs to be a clear split between observing a variant, and the evaluations/interpretations/implications related to the variant. The latter is a moment in time and the implications can change over time. Given that complexity, IMHO, CGWG took a thoughtful approach on how to modularize the two. 
We can discuss further in a meeting if needed.